communication/state.h

```diff
@@ -80,7 +80,7 @@ typedef enum
 typedef enum
 {
 	DESCENT_TO_SMALL_ALTITUDE,							///< First auto landing behavior
-	DESCENT_TO_GND										///< Second auto landing behavior, comes after DESCENT_TO_SMAL_ALTITUDE
+	DESCENT_TO_GND,										///< Second auto landing behavior, comes after DESCENT_TO_SMALL_ALTITUDE
```

Comment:
I did notice the change but the trailing comma doesn't have any effect on the enum so I figured I didn't need to change, but I can if you want.